spaces/domains.json

```
@@ -832,4 +832,6 @@
   "vote.lebo.finance": "lebo-finance.eth",
   "vote.cirip.io": "cirip.io", 
   "gov.infinitypad.com": "ipadgovernance.eth"
+  "vote.fndz.io": "fndzdao.eth"
+  
```

A comma is missing at the end of the line above your sub-domain. Please delete the extra line.
```suggestion
  "gov.infinitypad.com": "ipadgovernance.eth",
  "vote.fndz.io": "fndzdao.eth"
```